src/material/datepicker/datepicker-input.ts

```diff
@@ -183,5 +182,4 @@ export class MatDatepickerInput<D> extends MatDatepickerInputBase<D | null, D> {
   // Accept `any` to avoid conflicts with other directives on `<input>` that
   // may accept different types.
   static ngAcceptInputType_value: any;
-  static ngAcceptInputType_disabled: BooleanInput;
```

Comment:
was this intentional?

Reply:
Yes. I think either I missed removing it in a previous PR or we changed the lint rule to catch more things, because the `date-range` branch started failing the lint check after I rebased it. I decided to just fix the issue in this PR since it's going back into the same branch.